### ISSUE TITLE
myloader is crashing with invalid default value

### ISSUE THREAD

**Issue by sofire** [maxbube/mydumper#74]
```
/*!40101 SET NAMES binary*/;
/*!40014 SET FOREIGN_KEY_CHECKS=0*/;

CREATE TABLE `dede_member_space` (
  `mid` mediumint(8) unsigned NOT NULL DEFAULT '0',
  `pagesize` smallint(5) unsigned NOT NULL DEFAULT '10',
  `matt` smallint(6) NOT NULL DEFAULT '0',
  `spacename` varchar(50) NOT NULL DEFAULT '',
  `spacelogo` varchar(50) NOT NULL DEFAULT '',
  `spacestyle` varchar(20) NOT NULL DEFAULT '',
  `sign` varchar(100) NOT NULL DEFAULT '没签名',
  `spacenews` text,
  PRIMARY KEY (`mid`)
) ENGINE=InnoDB DEFAULT CHARSET=gbk;
```
** (myloader:9484): CRITICAL **: Error restoring xxx.dede_member_space from file xxx.dede_member_space-schema.sql.gz: Invalid default value for 'sign'

mydumper 0.9.3, built against MySQL 5.6.26
myloader 0.9.3, built against MySQL 5.7.19-17

**Comment by maxbube** [comment]
Hi @sofire 

thanks for the report and the test case, I will try to reproduce it.

**Comment by druud** [comment]
It should also add this to all foo.bar-schema.sql files:
/*!40103 SET TIME_ZONE='+00:00' */;
(unless --skip-tz-utc)
or it will easily fail on certain timestamp/datetime defaults like '1970-01-01 00:00:01'.

**Comment by davidducos** [comment]
The fix for this has been merged to master and will be released on 0.10.5
You will need to use --set-names=utf8
```
root@ubuntu-focal:/tmp/mydumper# ./mydumper -B issue74 -o issue74 --set-names=utf8
root@ubuntu-focal:/tmp/mydumper# ./myloader -B issue74_test -d issue74
root@ubuntu-focal:/tmp/mydumper# mysql issue74_test -e "show create table dede_member_space\G"
*************************** 1. row ***************************
       Table: dede_member_space
Create Table: CREATE TABLE `dede_member_space` (
  `mid` mediumint(8) unsigned NOT NULL DEFAULT '0',
  `pagesize` smallint(5) unsigned NOT NULL DEFAULT '10',
  `matt` smallint(6) NOT NULL DEFAULT '0',
  `spacename` varchar(50) NOT NULL DEFAULT '',
  `spacelogo` varchar(50) NOT NULL DEFAULT '',
  `spacestyle` varchar(20) NOT NULL DEFAULT '',
  `sign` varchar(100) NOT NULL DEFAULT '没签名',
  `spacenews` text,
  PRIMARY KEY (`mid`)
) ENGINE=InnoDB DEFAULT CHARSET=gbk
```
Closing this issue...
